app/serializers/post_serializer.rb

```diff
@@ -561,10 +561,10 @@ def user_status
   end
 
   def mentioned_users
```

Comment:
Related to [this comment](https://github.com/discourse/discourse/pull/19644/files#r1072940219), it seems like the `mentioned_users` attribute was added to support displaying user status. If user status is disabled, I don't think we need to return this attribute.